src/external_cli/cargo/install.rs

```diff
@@ -21,19 +27,41 @@ fn is_installed(program: &str) -> bool {
 pub(crate) fn if_needed(
     program: &str,
     package: &str,
+    package_version: Option<&str>,
     skip_prompts: bool,
     hidden: bool,
 ) -> anyhow::Result<bool> {
-    if is_installed(program) {
-        return Ok(false);
+    let mut prompt: Option<String> = None;
+
+    if let Some(stdout) = is_installed(program) {
+        let Some(package_version) = package_version else {
+            // If no `package_version` is specified and the program is installed,
+            // there is nothing to do.
+            return Ok(false);
+        };
+        // If `package_version` is specified, check if the installed version matches the desired
+        //`package_version`
```

Comment:
Nit: Perhaps we can adjust this comment a bit since the code is actually specific to wasm-bindgen right now.
Maybe we can explain that the version requirement has to match exactly here.